rxros_talker/src/rxros_talker.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright (c) 2019, ROSIN-project
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+1. Redistributions of source code must retain the above copyright notice, this
+list of conditions and the following disclaimer.
+
+2. Redistributions in binary form must reproduce the above copyright notice,
+this list of conditions and the following disclaimer in the documentation
+and/or other materials provided with the distribution.
+
+3. Neither the name of the copyright holder nor the names of its
+contributors may be used to endorse or promote products derived from
+this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+/**
+ * This tutorial demonstrates simple sending of messages over the ROS system 
+ * using the rxros interface
+ */
+
+#include <rxros/rxros.h>
+#include <std_msgs/String.h>
+
+using namespace rxcpp::operators;
+using namespace rxros::operators;
```

Comment:
> The type checker should detect an error if there is a clash (and then some measures can be taken).

It's not about "silent shadowing". It's about the pages of compiler errors that get spit out *when* it happens.

They're annoying and unnecessary.

Reply:
but the alternative is `a rxcpp::operators::| b` instead of 'a | b'. It simply sucks.

Reply:
I'm not suggesting you do that in this case.

Adding `using namespace rxcpp::operators` to the function scope in which you use the operators would be fine.

Thing is: besides what I wrote above, new users will copy whatever you are doing, as it's "like that in the example". With that in mind, I believe we should try to avoid introducing known anti-patterns.

Reply:
I completely agree.  I just did this (removed one import that had lesser syntactic impact, and localized the other).